docs/adr/0008-report-a-suspicious-activity.md

```diff
@@ -0,0 +1,44 @@
+# Reporting a Suspicious activity
+
+## Summary
+- We will create a Zendesk ticket when an activity is reported .
+- We need to publish an audit event to [TXMA] (https://govukverify.atlassian.net/wiki/spaces/TMA/pages/3414753375/Introduction+to+TxMA) which will be consumed by the fraud team.
+
+## Context
+
+Users need to be able to report sign in events from their ‘activity log’ which they do not recognise.
+
+We need to make sure that:
+
+- We collect data about the event the user is suspicious of, and about the account affected - ideally without the user having to enter anything
+
+- When a user submits a report, Zendesk will create a ticket containing the right information in the TICF 2nd line group.
+
+- An audit event is created in TXMA which can then be distributed out from there.
+
+- The user gets an immediate auto-response setting their expectations as to what will happen next
+
+## Decision
+We will set up an SNS topic for a report of a suspicious event, triggered upon a user submitting the contact form.
+
+The event will contain user details along with the events details .
```

Comment:
Could you add a sample payload of the event here please?

Reply:
Done